### PR TITLE
add docs for mentorguidelines.md

### PR DESCRIPTION
**Pull request by parispittman**
appreciate comments - this is a work in progress; there are some minor grammar errors that I see and white space issues that I will change after comments. 

/sig contributor-experience
/hold

### CODE REVIEW

File: mentoring/mentor-guide.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
+Mentor Guidelines
+=======
+*WIP*
+
+With over 37 special interest and working groups (SIGs/WGs), the Kubernetes organization is very large to navigate through and no one has all of the answers. Contributors rely on our docs, commenting on PRs/Issues, finding a mentor on their own, and any time they can interact with others to get up to speed. Whether it’s Slack, update meetings, or KubeCons; contributors want to discuss technical specifics, the landscape, or how to be a valuable member. Being a mentor can help accelerate someone's path to being more productive in our ecosystem.
```
Comment (justaugustus) [inline mentoring/mentor-guide.md]: > the Kubernetes organization is very large to navigate through and no one has all of the answers.
Feels a little weird in the mouth...

thoughts on:
> the Kubernetes ecosystem is vast and no one has all of the answers.
?

Comment (justaugustus) [reply mentoring/mentor-guide.md]: `KubeCon,` instead of `KubeCon;`

File: mentoring/mentor-guide.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+There are many ways you can be a mentor within the Kubernetes ecosystem. Listening and being generally helpful is an informal start but we’ve drawn together some programs that will be impactful. These programs also take your time into consideration for both parties - we know that it’s difficult to carve out steady time to mentor someone. Bonus: we set up the communication framework for you already so your expectations are set when you are ready to dig into mentoring.
```
Comment (justaugustus) [inline mentoring/mentor-guide.md]: `informal start,`*

Comment (justaugustus) [reply mentoring/mentor-guide.md]: `These programs also take your time into consideration for both parties` --> `These programs take both the mentee's AND the mentor's into consideration`

Comment (justaugustus) [reply mentoring/mentor-guide.md]: ` - we know that it’s difficult to carve out steady time to mentor someone` --> ` We know that it’s difficult to carve out steady time to mentor someone, and we appreciate that contribution to the Kubernetes community!`

Comment (justaugustus) [reply mentoring/mentor-guide.md]: Also, not sure sure that the bonus is really a bonus; more so just a facet of the program(s).

File: mentoring/mentor-guide.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+You are serving two roles - coach and advisor. You are a problem solver that doesn’t have all of the answers but will know the best resources to go to get the guidance that is needed. You are helping to create a culture of continuous improvement, collaboration, and feedback.
+
+Check out our [mentoring programs](mentoring) for more detailed descriptions. Please use [this form](https://goo.gl/forms/Y4MH6NGHe6OU5cZC3) to sign up to mentor.
+
+## Responsibilities and Expectations
+*for ALL programs*
+* Your position is to be a guide and not a teacher. If you have the time to teach someone the fundamentals of Go, please do, but know that this isn’t required. In this senario, you should be able to help identify the best resources for learning Go for the mentee.
```
Comment (justaugustus) [inline mentoring/mentor-guide.md]: `position` --> `goal`

Comment (justaugustus) [reply mentoring/mentor-guide.md]: scenario*

Comment (justaugustus) [reply mentoring/mentor-guide.md]: btw: `make verify` failed to pick up the misspelling...

File: mentoring/mentor-guide.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+## Responsibilities and Expectations
+*for ALL programs*
+* Your position is to be a guide and not a teacher. If you have the time to teach someone the fundamentals of Go, please do, but know that this isn’t required. In this senario, you should be able to help identify the best resources for learning Go for the mentee. 
+* Issue Grooming. Most levels of the [membership ladder](/community-membership.md) will require some form of issue grooming from you. Examples:
+*  picking issues that don’t span multiple SIGs for new contributors
+*  or they do because they are learning to be a better approver
+*  labeling that will help them identify clear issues that need their attention
+* Make introductions to resources and people for the mentee - this is sponsorship which is now equally as important as mentorship.
+* Help contributor be productive.
+* Patience and empathy.  Through repetition, people learn, and as a mentor, we cannot expect a mentee to meet all of our expectations.  Through the removal of expectation and bias, with a sprinkle of compassion, we can gain more patience.
```
Comment (justaugustus) [inline mentoring/mentor-guide.md]: `, with` --> `, and with`

File: mentoring/mentor-guide.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+* Have some fun and learn together!
+
+
+### Responsibilities by each program
+*Group mentoring*
+
+You’ll need to be:
+*  On slack at least 1-2 hours a week fielding questions.
```
Comment (justaugustus) [inline mentoring/mentor-guide.md]: `slack` --> `Slack`

File: mentoring/mentor-guide.md [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+*  picking issues that don’t span multiple SIGs for new contributors
+*  or they do because they are learning to be a better approver
+*  labeling that will help them identify clear issues that need their attention
+* Make introductions to resources and people for the mentee - this is sponsorship which is now equally as important as mentorship.
+* Help contributor be productive.
+* Patience and empathy.  Through repetition, people learn, and as a mentor, we cannot expect a mentee to meet all of our expectations.  Through the removal of expectation and bias, with a sprinkle of compassion, we can gain more patience.
+* Have some fun and learn together!
+
+
+### Responsibilities by each program
```
Comment (justaugustus) [inline mentoring/mentor-guide.md]: Maybe link out to the individual pages instead?

File: mentoring/mentor-guide.md [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+
+You are serving two roles - coach and advisor. You are a problem solver that doesn’t have all of the answers but will know the best resources to go to get the guidance that is needed. You are helping to create a culture of continuous improvement, collaboration, and feedback.
+
+Check out our [mentoring programs](mentoring) for more detailed descriptions. Please use [this form](https://goo.gl/forms/Y4MH6NGHe6OU5cZC3) to sign up to mentor.
+
+## Responsibilities and Expectations
+*for ALL programs*
+* Your position is to be a guide and not a teacher. If you have the time to teach someone the fundamentals of Go, please do, but know that this isn’t required. In this senario, you should be able to help identify the best resources for learning Go for the mentee. 
+* Issue Grooming. Most levels of the [membership ladder](/community-membership.md) will require some form of issue grooming from you. Examples:
+*  picking issues that don’t span multiple SIGs for new contributors
```
Comment (justaugustus) [inline mentoring/mentor-guide.md]: I think the indented bullets got weird here.

File: mentoring/mentor-guide.md [hunk 8]
```diff
@@ -0,0 +1,101 @@
...
+You’ll need to be:
+*  On slack at least 1-2 hours a week fielding questions.
+*  Available for a group bi-weekly standup to check the progress of the mentees. 
+*  Helpful with building comradery amongst the group - this can be as simple as curating conversation in slack.
+
+Mentee’s will be asked to take responsibility for their growth and improvement as it's a semi-structured self paced program; however, you are there to help guide them by sharing your methodologies, resources, tips, and experiences.
+
+*The 1:1 Hour*
+* Participate in a one hour a month session via zoom video conference; no more than 3 hours/times and that is opt-in.
+* Depending on the activities that you’ve selected as comfortable in helping with, the mentee may ask:
```
Comment (justaugustus) [inline mentoring/mentor-guide.md]: I think the indented bullets got weird here.

File: mentoring/mentor-guide.md [hunk 9]
```diff
@@ -0,0 +1,101 @@
...
+
+### Responsibilities by each program
+*Group mentoring*
+
+You’ll need to be:
+*  On slack at least 1-2 hours a week fielding questions.
+*  Available for a group bi-weekly standup to check the progress of the mentees. 
+*  Helpful with building comradery amongst the group - this can be as simple as curating conversation in slack.
+
+Mentee’s will be asked to take responsibility for their growth and improvement as it's a semi-structured self paced program; however, you are there to help guide them by sharing your methodologies, resources, tips, and experiences.
```
Comment (justaugustus) [inline mentoring/mentor-guide.md]: Mentees*

Comment (justaugustus) [reply mentoring/mentor-guide.md]: `, self-paced program. However,`*

File: mentoring/mentor-guide.md (hunk 1 above)
Comment (parispittman) [reply mentoring/mentor-guide.md]: ty!

re: becoming README: I'd like the keep the README that is the list of programs we have and have this be just the mentor guide.

File: mentoring/mentor-guide.md (hunk 2 above)
Comment (parispittman) [reply mentoring/mentor-guide.md]: ty!

**Comment by parispittman** [comment]
@tpepper @justaugustus - want to take a final peek for me?

**Comment by tpepper** [comment]
Will do

**Comment by cblecker** [comment]
/lgtm
Testing something :)

File: mentoring/mentor-guide.md [hunk 10]
```diff
@@ -0,0 +1,100 @@
+Mentor Guidelines
+=======
+
+With over 37 special interest and working groups (SIGs/WGs), the Kubernetes ecosystem is vast and no one has all of the answers. Contributors rely on our docs, commenting on PRs/Issues, finding a mentor on their own, and any time they can interact with others to get up to speed. Whether it’s Slack, update meetings, or KubeCons, contributors want to discuss technical specifics, the landscape, or how to be a valuable member. Being a mentor can help accelerate someone's path to being more productive in our ecosystem.
+
+There are many ways you can be a mentor within the Kubernetes ecosystem. Listening and being generally helpful is an informal start, but we’ve drawn together some programs that will be impactful. These programs take the mentee's AND mentor's time into consideration. We know that it’s difficult to carve out steady time to mentor someone, and we appreciate that contributor to the Kubernetes community! The Bonus: we set up the communication framework for you already so your expectations are set when you are ready to dig into mentoring.
```
Comment (tpepper) [inline mentoring/mentor-guide.md]: "...that contributor..." --> "...that contribution..."

File: mentoring/mentor-guide.md [hunk 11]
```diff
@@ -0,0 +1,100 @@
...
+
+Check out our [mentoring programs](README.md) for more detailed descriptions. Please use [this form](https://goo.gl/forms/Y4MH6NGHe6OU5cZC3) to sign up to mentor.
+
+## Responsibilities and Expectations
+*for ALL programs*
+* Your goal is to be a guide and not a teacher. If you have the time to teach someone the fundamentals of Go, please do, but know that this isn’t required. In this scenario, you should be able to help identify the best resources for learning Go for the mentee. 
+* Issue Grooming. Most levels of the [membership ladder](/community-membership.md) will require some form of issue grooming from you. Examples:
+*  picking issues that don’t span multiple SIGs for new contributors
+*  or they do because they are learning to be a better approver
+*  labeling that will help them identify clear issues that need their attention
```
Comment (tpepper) [inline mentoring/mentor-guide.md]: these three bullets didn't indent.  Add space before the star.

File: mentoring/mentor-guide.md [hunk 12]
```diff
@@ -0,0 +1,100 @@
...
+*[Group mentoring](group-mentoring.md)*
+
+You’ll need to be:
+*  On Slack at least 1-2 hours a week fielding questions.
+*  Available for a group bi-weekly standup to check the progress of the mentees. 
+*  Helpful with building comradery amongst the group - this can be as simple as curating conversation in slack.
+
+Mentees will be asked to take responsibility for their growth and improvement as it's a semi-structured self-paced program; however, you are there to help guide them by sharing your methodologies, resources, tips, and experiences.
+
+*[The 1:1 Hour](the1-on-1.md)*
```
Comment (tpepper) [inline mentoring/mentor-guide.md]: If the other PR's commit is not inbound, this link will be broken.  Actually that one links to this.  So ideally they merge in very close succession.

File: mentoring/mentor-guide.md [hunk 13]
```diff
@@ -0,0 +1,100 @@
...
+* Participate in a one hour a month session via zoom video conference; no more than 3 hours/times and that is opt-in.
+* Depending on the activities that you’ve selected as comfortable in helping with, the mentee may ask:
+*  To pair program with you.
+*  Ask you an hours worth of questions (AMA). 
+*  Do code reviews/docs reviews.
+*  Discuss their path so far and ask for input/feedback.
+*  Code base tour of a certain area
+*  Or a mix - it’s up to the mentee with how they’d like to spend the time and your skills/experience.
```
Comment (tpepper) [inline mentoring/mentor-guide.md]: Another set of intended indents here that didn't indent.

File: mentoring/mentor-guide.md [hunk 14]
```diff
@@ -0,0 +1,100 @@
...
+* You'll fill out a form once a month to list your availibility.
+
+*[Outreachy](https://www.outreachy.org)*
+* Being a mentor can take anywhere from 2-5 hours a week depending on the scope of the project that the intern(s) work on. 
+* Before the program issue grooming/project planning activities:
```
Comment (tpepper) [inline mentoring/mentor-guide.md]: suggest:

```
Before the program:
 * issue grooming/project planning activities
 ...
```

File: mentoring/mentor-guide.md [hunk 15]
```diff
@@ -0,0 +1,100 @@
...
+  * For the Outreachy application process that they host [here](https://www.outreachy.org/apply/). All interns that are interested in the project must make one contribution as part of their application. Label them with a good first time issue.
+* During the program:
+  * The time above is split up between slack, 1:1 pair programming and standups.
+  * Provide official feedback during intervals to Outreachy and the intern.
+  * Have read and understand the [rules and guidelines](https://www.outreachy.org/mentor/) around mentoring for Outreachy
```
Comment (tpepper) [inline mentoring/mentor-guide.md]: only bullet here without a trailing period.

File: mentoring/mentor-guide.md [hunk 16]
```diff
@@ -0,0 +1,100 @@
...
+
+CNCF will financially support one Outreachy intern for Kubernetes; if you would like to participate and we already have one, assistance with securing a corporate sponsor for the budget (6.5k) may be needed.
```
Comment (tpepper) [inline mentoring/mentor-guide.md]: This is reading a little odd now to me.  Suggest: "...for the additional budget..."

File: mentoring/mentor-guide.md [hunk 17]
```diff
@@ -0,0 +1,100 @@
...
+  * Evaluate the final proposals and select the student you want to mentor.
+* During the program:
+  * Set development and communication plans with your mentee. Make sure you establish realistic work objectives and timeline expectations.
+  * Help them get integrated into the community and encourage them to participate.
+  * Provide guidance such as pointers to useful documentation, code reviews, etc.
+  * Set aside time for a weekly sync via Slack or video call, whichever you prefer.
+  * Collect weekly status reports. They are important in making sure that time-line slippages and scope creep are addressed in a proactive manner.
+  * Keep track of the mentee's progress and keep them informed about their status. Make sure you deliver feedback early.
+  * Be responsive to questions - usual expectation is to reply within 24 hours (occasionally under 36 hours is ok).
+  * There will be three evaluations of your mentee's work. Submit them timely. The evaluation forms take hardly 10-15 min to fill.
```
Comment (tpepper) [inline mentoring/mentor-guide.md]: spell out "minutes"

File: mentoring/mentor-guide.md [hunk 18]
```diff
@@ -0,0 +1,100 @@
...
+  * Recommend next steps to your mentee in the current project or wherever you think they might fit in our community.
+
+A former GSoC intern for Kubernetes is now leading this for us - mentoring works! Thanks @nikhita!
+
+*[Meet Our Contributors - Mentors On Demand](meet-our-contributors.md)*
+* One hour a month on a streaming YouTube show answering questions about the contribution process, your experience in open source, and other questions that people would normally ask to a mentor.
+* Typically a one hour a quarter time commitment is expected; this is an opt-in sign up.
+* You will need to be on the call 5 mins early to test audio quality and make sure we are good to go for streaming.
+
+### Benefits of being a mentor
```
Comment (tpepper) [inline mentoring/mentor-guide.md]: These trailing three sections now feel like a bit of an afterthought, where perhaps if they were at the start of the doc they encourage folks to keep reading and engage?

File: mentoring/mentor-guide.md (hunk 12 above)
Comment (parispittman) [reply mentoring/mentor-guide.md]: This is doc is pending right now

**Comment by parispittman** [comment]
@tpepper - thank you for the review. I made the changes.

File: mentoring/mentor-guide.md (hunk 17 above)
Comment (tpepper) [reply mentoring/mentor-guide.md]: Deliberately skipped?  Either way just a suggestion for ease of readability.

**Comment by tpepper** [comment]
One question remaining ^^, but it's a nit really.

/lgtm

File: mentoring/mentor-guide.md (hunk 17 above)
Comment (parispittman) [reply mentoring/mentor-guide.md]: Oh - I changed another instance and must have missed this one

**Comment by cblecker** [comment]
/approve

🎉 🏆

**Pull request merged**